dpctl/sycl_core.pyx

```diff
@@ -132,6 +133,21 @@ cdef class SyclQueue:
     cdef DPPLSyclQueueRef get_queue_ref (self):
         return self.queue_ptr
 
+    cpdef memcpy (self, dest, src, int count):
+        cdef void *c_dest, *c_src
```

Comment:
Cython generates warning here. It requires declarations on separate lines:

```
cdef void * c_dest
cdef void * c_src
```

Reply:
Fixed.